run.go

```diff
@@ -274,7 +276,11 @@ func supportsConcurrency(format string) bool {
 	switch format {
 	case "progress", "junit":
 		return true
-	case "events", "pretty", "cucumber":
+	case "events":
```

Comment:
Was this not working before?

Reply:
it was, but I made this change to have easier merges later,`supportsConcurrency` will be removed as soon as all formatters updates are merged

Reply:
Works for me :)